tests/Benchmarks/System.Text.Primitives/PrimitiveParserInt16PerfTests.cs

```diff
@@ -13,56 +13,6 @@ namespace System.Text.Primitives.Benchmarks
 {
     public partial class PrimitiveParserPerfTests
     {
-        private static readonly string[] s_Int16TextArray = new string[13]
-        {
-            "21474",
-            "2",
-            "-21474",
-            "31484",
-            "-21",
-            "-2",
-            "214",
-            "2147",
-            "-2147",
-            "-9345",
-            "9345",
-            "1000",
-            "-214"
-        };
-
-        public IEnumerable<object> PrimitiveParserByteSpanToInt16_VariableLength_Arguments()
-            => s_Int16TextArray.Select(text => new Utf8ByteArrayArgument(text));
-        
-        public IEnumerable<object> PrimitiveParserByteSpanToInt16_Arguments()
-        {
-            yield return new Utf8ByteArrayArgument("10737"); // standard parse
-            yield return new Utf8ByteArrayArgument("32767"); // max value
-            yield return new Utf8ByteArrayArgument("0"); 
-            yield return new Utf8ByteArrayArgument("-32768"); // min value
-            yield return new Utf8ByteArrayArgument("-000000000000000000001235abcdfg");
-            yield return new Utf8ByteArrayArgument("-2147abcdefghijklmnop");
-        }
-
-        [Benchmark]
-        [ArgumentsSource(nameof(PrimitiveParserByteSpanToInt16_Arguments))]
-        public bool PrimitiveParserByteSpanToInt16(Utf8ByteArrayArgument text) 
```

Comment:
Same here. Keep this particular test and update `PrimitiveParserByteSpanToInt16_Arguments` to match the thai digits.